readthedocs/source/orb/restendpoints/did-web.md

```diff
@@ -0,0 +1,72 @@
+# DID Web File Endpoint
+
+The [DID Web](https://w3c-ccg.github.io/did-method-web/#read-resolve) file endpoint is used for resolution of did:web method documents by universal resolver and other did:web resolvers. This endpoint will serve did document only not did resolution result.
```

Comment:
Should capitalize DID: ... serve the DID document, ... not the DID resolution ...

Reply:
fixed @bstasyszyn